quickstart/src/index.js

```diff
@@ -168,16 +197,19 @@ document.getElementById('button-preview').onclick = function() {
     ? Promise.resolve(previewTracks)
     : Video.createLocalTracks();
 
-  localTracksPromise.then(function(tracks) {
-    window.previewTracks = previewTracks = tracks;
-    var previewContainer = document.getElementById('local-media');
-    if (!previewContainer.querySelector('video')) {
-      attachTracks(tracks, previewContainer);
+  localTracksPromise.then(
+    function(tracks) {
+      window.previewTracks = previewTracks = tracks;
+      var previewContainer = document.getElementById('local-media');
+      if (!previewContainer.querySelector('video')) {
+        attachTracks(tracks, previewContainer);
+      }
+    },
+    function(error) {
+      console.error('Unable to access local media', error);
+      log('Unable to access Camera and Microphone');
```

Comment:
@PikaJoyce I don't think this styling change is necessary.

Reply:
Sorry. This was done by my auto-matter.